recipe/0103-poetry-reading-annotations/annotations.json

```diff
@@ -0,0 +1,48 @@
+{
+   "@context":"http://iiif.io/api/presentation/3/context.json",
+   "id":"{{ id.url }}",
+   "type":"AnnotationPage",
+   "items":[
+      {
+         "id":"{{ id.path }}/canvas/annotation1",
+         "type":"Annotation",
+         "motivation":"commenting",
```

Comment:
Per Rob's comment "Is "her kind" a transcription of the spoken content? If so then it should be motivation supplementing, not commenting. See: https://iiif.io/api/presentation/3.0/#35-values"

This should be:
```
"motivation":["supplementing"]
```

Reply:
but should be an array of values.

Reply:
Comment.  Pretty sure our academic partner calls this "paratext", which I wouldn't call transcription.

Reply:
Sorry I'm wrong here it should be a string not an array...